lib/dor/text_extraction/abbyy/results.rb

```diff
@@ -33,7 +43,7 @@ def output_docs
           {}.tap do |structural|
             output_docs.each do |doc|
               doc_type = doc.xpath('@ExportFormat').text.downcase
-              structural[doc_type] = File.join(doc.xpath('@OutputLocation').text, doc.xpath('FileName').text)
+              structural[doc_type] = local_output_path(doc.xpath('@OutputLocation').text, doc.xpath('FileName').text)
```

Comment:
This is needed to convert the Windows location reported by Abbyy to the local Unix location